app/objects/c_ability.py

```diff
@@ -13,12 +14,12 @@
 
 class AbilitySchema(ma.Schema):
     ability_id = ma.fields.String()
-    tactic = ma.fields.String(missing=None)
+    tactic = ma.fields.String(required=True)
```

Comment:
My reasoning was that on the old UI, tactic/executors were required fields in order to save an ability. I can remove these changes if that's preferred though, just so there isn't the possibility of related issues. I could just validate the data received on API requests instead using `_validate_ability_data()` in `ability_api_manager.py`

Reply:
This has been updated so that tactic/executors are no longer required fields - instead, `ability_api_manager.py` will validate received API data.

The only change I made to the schema is removing missing=None for the executors list; that causes an iteration error (without much detail) within the marshmallow decorator when an empty `executors` list is passed in a payload.